src/tools/rest-frontend/resources/grunt-tasks/grunt-create-website-news-rss.js

```diff
@@ -45,7 +45,7 @@ module.exports = function (grunt) {
             // filter guid from news post
             var guid = (new RegExp(self.data.regex.guid.pattern, self.data.regex.guid.flags)).exec(content);
             if (guid === null || guid.length < 2) {
-                grunt.log.error('Could not find `guid` in news, cannot create RSS file.');
```

Comment:
Maybe a warning here, too? Just ignoring news that do not have a guid seems rude.